packages/paste-website/src/pages/patterns/empty-states/index.mdx

```diff
@@ -0,0 +1,444 @@
+      </Heading>
+    </Card>
+  </Column>
+  <Column>
+    <Card>
+      <Heading as="h3" variant="heading40" marginBottom="space0">
+        <Anchor href="../illustrations">Illustrations (optional)</Anchor>
+      </Heading>
+    </Card>
+  </Column>
+</Grid>
+
+```jsx
+// import all ingredients for the empty state patterns
+import {​Card} from '@twilio-paste/core/card';
+import {Heading} from '@twilio-paste/core/heading';
+import {Paragraph} from '@twilio-paste/core/paragraph';
+import {Button} from '@twilio-paste/core/button';
+import {​Anchor} from '@twilio-paste/core/anchor';
+```
+
+## Usage
+
+### General
+
+This pattern is used to explain to a customer what they should do next if content is either unavailable or can't be displayed.
+
+Empty states should:
+
+- **Be clear:** If there's a path forward, explain it as concisely as possible. Include an explanation for why a customer is seeing the empty state if it helps them better understand what to do next.
+- **Give direction and educate the customer:** Let customers know what steps they can take to move forward in a flow. Guiding them to "Create an alpha sender" is more helpful than saying "No alpha senders".
+- **Be encouraging and inspire confidence:** Use a [friendly and approachable tone](/content/voice-and-tone#encouraging) to ensure a positive experience. Never make customers feel unsuccessful or guilty because they're seeing an empty state. Even when they arrive at a page "in error" (for example, on a page they don't have permissions to), encouraging messaging and an explanation of possible solutions can give customers confidence that they're heading in the right direction.
+
+In most cases, treat an empty state as an informational state of the entire page, and not a state that simply replaces a part of the page that should otherwise be populated. When you have text explaining what a page is for, try working it into the empty state card.
+
+When a page is in an empty state, its call-to-action (CTA) should appear in the empty state card, not near the page heading like in the [Create pattern](/patterns/create/). Exceptions can be made in cases like some "No results found" scenarios, when a CTA is already on the page on load. The visually focus of hiding the CTA that's outside of the card could draw too much of the customer's attention away from their primary goal.
+
+<Box marginBottom="space70">
+  <iframe
+    src="https://codesandbox.io/embed/empty-state-with-immediate-next-step-bmxe1?fontsize=14&hidenavigation=1&theme=dark&view=preview"
+    style="width:100%; height:350px; border:0; border-radius: 4px; overflow:hidden;"
+    title="Empty state requiring a customer to wait"
+    allow="accelerometer; ambient-light-sensor; camera; encrypted-media; geolocation; gyroscope; hid; microphone; midi; payment; usb; vr; xr-spatial-tracking"
+    sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
+  ></iframe>
+</Box>
```

Comment:
Did using SVGs not work? Should we include a callout that tells people to use PNGs for now?

Reply:
We can update the codesandboxes with the svgs. I used pngs to just get something up there for now.

Reply:
@serifluous I took at this, and we'll need to do some more work to get svgs working in our codesandboxes. The svgs are also HUGE: ~400kb vs ~60kb for the pngs. I think we should leave them as pngs for now. Downstream teams will most likely have a way to handle svg loading already.

Reply:
Until we have themeable illustrations I think PNGs are fine.